src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpUnboundIdentifiersDiagnosticAnalyzer.cs

```diff
@@ -55,7 +55,7 @@ protected override bool ConstructorDoesNotExist(SyntaxNode node, SymbolInfo info
             {
                 for (int i = 0; i < constructor.Parameters.Length; i++)
                 {
-                    var typeInfo = model.GetTypeInfo(args[i].Expression);
+                    var typeInfo = model.GetTypeInfo(args[i].Expression != null ? args[i].Expression : args[i].Declaration.Type);
```

Comment:
sure

---
In reply to: [68112502](https://github.com/dotnet/roslyn/pull/12159#discussion_r68112502) [](ancestors = 68112502)